components/dropdown/dropdown-content-mixin.js

```diff
@@ -475,6 +475,8 @@ export const DropdownContentMixin = superclass => class extends LocalizeCoreElem
 
 	async __openedChanged(newValue) {
 
+		// DE44538: wait for dropdown content to fully render,
```

Comment:
Adding this comment as a follow-up to https://github.com/BrightspaceUI/core/pull/1523